### ISSUE TITLE
Stored checkpoints are not restored correctly

### ISSUE THREAD

**Issue by meesoft** [SciSharp/TensorFlow.NET#522]
If I store a checkpoint after training for a long time with either Tensorflow/Python or TensorFlow.NET and then restore it in TensorFlow.NET then it doesn't give the correct results.

I've trained (in TensorFlow.NET) to e.g. a loss value of ~1.3 and if I then save a checkport and restore it in TensorFlow.NET I get a loss value of around 2.0. If I then take the same checkpoint from TensorFlow.NET and load in a Python implementation I get the expected 1.3 loss.

Did anyone else see something like that? It is this image processing problem I'm working on: https://github.com/meesoft/DnCNN-tensorflow
It is basically a lot of convolution layers with batch normalization.

**Comment by raghavendraLhci** [comment]
Hi.. @meesoft  can you please tell me how did you load the ckpt file in c# that is stored using python. Thank you.
